### DISCUSSION TITLE
Launchbar action not working with 1.4.1

### DISCUSSION THREAD

**Discussion by DugRay** [ttscoff/bunch-docs#98] [Q&A]
Is it just me (it could definitely be me), or is the [Launchbar action](https://bunchapp.co/docs/integration/launchbar-and-alfred/) broken with the latest version of Bunch? Thanks! -- Doug

**Comment by ttscoff** [comment]
Are you on the latest version of the LaunchBar action? It had some major 
updates with the release of 1.4.

-Brett

**Reply by DugRay** [reply]
Downloaded 2.1.0, which seems to be the latest version. It shows in the Action Editor but no action(s) shows up in the index. I have other 3rd-party and some simple actions of my own and they work OK. I've reindexed, restarted, rebooted to no avail. Not that big a deal, though. I've just been using the menu bar.

**Comment by myildi** [comment]
Hello,
I am testing Bunch, too, and have just installed the LaunchBar action (v2.01, the one available on Brett's website today). When I invoke LB and type BU, Bunch Beta is indicated, but when I click Return, it is just launched without anything else shown. My bunches are under Dropbox in /Users/myildi/Library/CloudStorage/Dropbox/Documents/bunches 
I added this folder to be indexed in case it helps, but everything has stayed the same.

**Reply by ttscoff** [reply]
Yeah, I think the LB action is broken and I haven't looked at it in quite a while. I'll try to find some time to figure out what's wrong.

**Comment by myildi** [comment]
Le 8 sept. 2024 à 19:20, Brett Terpstra ***@***.***> a écrit :﻿
Yeah, I think the LB action is broken and I haven't looked at it in quite a while. I'll try to find some time to figure out what's wrong.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>
Thank you very much Brett!
